db/migrate/20191028115549_create_application_qualification.rb

```diff
@@ -0,0 +1,15 @@
+class CreateApplicationQualification < ActiveRecord::Migration[6.0]
+  def change
+    create_table :application_qualifications do |t|
+      t.references :application_form, null: false, foreign_key: { on_delete: :cascade }
+
+      t.string :type, null: false
+      t.string :qualification_type, null: false
```

Comment:
I guess this is a type in addition to the `degree`/`gcse` categorisation. Depending on whether it has a short fixed list of values it may be worth considering using an enum.

Reply:
Following Duncan's suggestion we have decided to go with the enum option and a generic `ApplicationQualification`.